cg/cli/workflow/mip_rna/base.py

```diff
@@ -86,14 +84,13 @@ def run(
     start_with: str = None,
 ):
     """Run the analysis for a case"""
-    tb_api = context.obj["tb"]
     rna_api = context.obj["rna_api"]
-    case_obj = context.obj["db"].family(case_id)
+    case_obj = rna_api.db.family(case_id)
 
     if not case_exists(case_obj, case_id):
         context.abort()
 
-    if tb_api.analyses(family=case_obj.internal_id, temp=True).first():
+    if rna_api.tb.analyses(family=case_obj.internal_id, temp=True).first():
```

Comment:
I think we should reuse the functions from the mip-dna/base here for anlyses. That shoud use the same logic

Reply:
Unfortunately there are differences... mip-dna will check if all samples are dna in ClinicalDB. Same goes for other commands, there are slight differences. Maybe a bit out of scope of this PR to harmonize them

Reply:
Yes. Let's do it another PR